config/example/node.yaml

```diff
@@ -1,159 +1,159 @@
+    endpoint: s01.neofs.devenv:8080  # endpoint for gRPC server
     tls:
-      enabled: true
-      certificate: /path/to/cert
-      key: /path/to/key
+      enabled: true  # use TLS for a gRPC connection
+      certificate: /path/to/cert  # path to TLS certificate
+      key: /path/to/key  # path to TLS key
 
   1:
-    endpoint: s02.neofs.devenv:8080
+    endpoint: s02.neofs.devenv:8080  # endpoint for gRPC server
     tls:
-      enabled: false
+      enabled: false  # use TLS for a gRPC connection
 
 control:
-  authorized_keys:
+  authorized_keys:  # list of raw hex public keys that have rights to use the Control Service
     - 035839e45d472a3b7769a2a1bd7d54c4ccd4943c3b40f547870e83a8fcbfb3ce11
     - 028f42cfcb74499d7b15b35d9bff260a1c8d27de4f446a627406a382d8961486d6
-  grpc:
+  grpc:  # endpoints that are listened by the Control Service
     endpoint: 127.0.0.1:8090
 
-contracts:
+contracts:  # side chain NEOFS contract script hashes
   balance: 5263abba1abedbf79bb57f3e40b50b4425d2d6cd
   container: 5d084790d7aa36cea7b53fe897380dab11d2cd3c
   netmap: 0cce9e948dca43a6b592efe59ddb4ecb89bdd9ca
   reputation: 441995f631c1da2b133462b71859494a5cd45e90
   proxy: ad7c6b55b737b696e5c82c85445040964a03e97f
 
 morph:
-  dial_timeout: 30s
-  disable_cache: true
-  rpc_endpoint:
+  dial_timeout: 30s  # timeout for side chain NEO RPC client connection
+  disable_cache: true  # do not use TTL cache for side chain GET operations
+  rpc_endpoint:  # side chain NEO RPC endpoints; are shuffled and used one by one until the first success
     - https://rpc1.morph.fs.neo.org:40341
     - https://rpc2.morph.fs.neo.org:40341
-  notification_endpoint:
+  notification_endpoint:  # side chain NEO RPC notification endpoints
     - wss://rpc1.morph.fs.neo.org:40341/ws
     - wss://rpc2.morph.fs.neo.org:40341/ws
   notary:
-    deposit_amount: 100000001
-    deposit_duration: 1001
+    deposit_amount: 100000001  # notary deposit GAS amount
+    deposit_duration: 1001  # notary deposit expiration in blocks
 
-mainchain:
-  dial_timeout: 30s
-  rpc_endpoint:
+mainchain:  # DEPRECATED section, is not used and not read
+  dial_timeout: 30s  # timeout for main chain NEO RPC client connection
+  rpc_endpoint:  # main chain NEO RPC endpoints; are shuffled and used one by one until the first success
     - https://rpc1.n3.nspcc.ru:30341
     - https://rpc2.n3.nspcc.ru:30341
 
 apiclient:
-  dial_timeout: 15s
+  dial_timeout: 15s  # timeout for NEOFS API client connection
 
 policer:
-  head_timeout: 15s
+  head_timeout: 15s  # timeout for the Policer HEAD remote operation
 
 replicator:
-  put_timeout: 15s
+  put_timeout: 15s  # timeout for the Replicator PUT remote operation
 
 object:
   put:
-    pool_size_remote: 100
-    pool_size_local: 101
+    pool_size_remote: 100  # number of async workers for remote PUT operations
+    pool_size_local: 101  # number of async workers for local PUT operations
 
 storage:
-  shard_num: 2
+  shard_num: 2  # total shards number
```

Comment:
I think "total number of shards" would sound better

Reply:
changed